services/app/src/components/gameFooter/chess-board-pieces.css

```diff
@@ -21,15 +21,15 @@
 
 .piece-field-selected {
   background-color: var(--chess-tile-light);
-  border-color: #6547FD;
+  border-color: var(--game-light-purple);
 }
 
 .piece-field.disabled {
   cursor: not-allowed;
 }
 
 .figure-img {
-  height: 35px;
+  height: clamp(30px , 3.5vh, 35px);
```

Comment:
Yes, it is, but we still need to set height for `figure-img`, because svgs can overflow its parent. And we need a height to make those svgs look consistent.

Reply:
You should be able to make it fit into parent container.